charts/common/templates/_deployment.yaml

```diff
@@ -293,6 +293,12 @@ spec:
             - name: CONSORTIUM_ENABLED
               value: {{ .Values.consortiumEnabled | quote }}
           {{- end }}
+          {{- if .Values.rtrEnabled }}
+            - name: LEGACY_TOKEN_TENANTS
```

Comment:
Is during render value sets to empty or it is better to set value: '' to avoid missing key or smth like